pymilvus/bulk_writer/local_bulk_writer.py

```diff
@@ -76,20 +95,27 @@ def commit(self, **kwargs):
         x = Thread(target=self._flush, args=(call_back,))
```

Comment:
we don't need to flush every append right?
only if buffer is large than expect size? then we change to a new buffer and flush the old buffer async